Decred Wallet/Features/Settings/SettingsController.swift

```diff
@@ -159,6 +201,24 @@ class SettingsController: UITableViewController  {
         tableView.reloadData()
     }
     
+    override func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        
+        let sectionName: String
```

Comment:
Instead of declaring this variable, and assigning to it, you can return from each case directly - fewer lines of code and more direct.